kubernetes/loculus/values.yaml

```diff
@@ -1421,6 +1426,72 @@ defaultOrganisms:
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/sars-cov-2/ORF9b.fasta]]"
         - name: "S"
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/sars-cov-2/S.fasta]]"
+  dummy-organism-with-files:
+    schema:
+      image: "https://cdn.who.int/media/images/default-source/mca/mca-covid-19/coronavirus-2.tmb-1920v.jpg?sfvrsn=4dba955c_19"
+      organismName: "Test organism (with files)"
+      nucleotideSequences: []
+      submissionDataTypes:
+        consensusSequences: false
+        files:
+          enabled: true
+          fields:
+            - name: raw_reads
+              multipleFiles: false
+      files:
+        - name: raw_reads
+          multipleFiles: false
```

Comment:
We should get rid of this duplication here, see comment above.

Reply:
Adressed in comment above, let's keep the discussion there. resolving.